File: 프로그래머스/unrated/181862. 세 개의 구분자/세 개의 구분자.py
def solution(myStr):
    #ver 1
    #removedA = myStr.split('a')
    #removedB = sum([x.split('b') for x in removedA if x ], [])
    #removedC = sum([x.split('c') for x in removedB if x ], [])
    #return [x for x in removedC if x] if len([x for x in removedC if x]) else ["EMPTY"]
    
    #ver 2
    remove = ['a', 'b', 'c']
    removed = ''.join(list(map(lambda el : '/' if el in remove else el, list(myStr)))).split('/')
    return [x for x in removed if x] if len([x for x in removed if x]) else ["EMPTY"]

File: 프로그래머스/unrated/181862. 세 개의 구분자/세 개의 구분자.js
function solution(myStr) {
    const remove = ['a', 'b', 'c'];
    const separate =  myStr.split('').map(el => remove.includes(el) ? el = '/' : el).join('').split('/').filter(el => el !== '')
    return separate.length === 0 ? ["EMPTY"] : separate
    
    //ver 2
    // const arr = myStr.split(/[abc]/).filter(el => el);
    // return arr.length == 0 ? ["EMPTY"] : arr;
}